http-default-accounts-fingerprints-nndefaccts.lua

```diff
@@ -5846,8 +5846,16 @@ table.insert(fingerprints, {
   },
   login_check = function (host, port, path, user, pass)
     local qstr = url.build_query({t=os.date("!%a, %d %b %Y %H:%M:%S GMT")})
+
+    local creds_alt = {username = user, password = pass, digest = false}
+    local resp_alt = http_post_simple(host, port,
+                                 url.absolute(path, "form-submit/auth.htm"),
+                                 {auth=creds_alt}, "")
+
     return try_http_auth(host, port, url.absolute(path, "auth.htm?" .. qstr),
                         user, pass, false)
+                        or
+                        (resp_alt.status == 200 and get_cookie(resp_alt,"session"))
```

Comment:
The concept of trying different login sequences for the same `target_check` is not desirable, as it generates unnecessary traffic. A cleaner way would be to carve out another fingerprint.

Can you share the actual HTTP trace, as described in the [README](https://github.com/nnposter/nndefaccts#fingerprint-contributions)?